### ISSUE TITLE
Import Database Fails if DEFAULT is set on any property

### ISSUE THREAD

**Issue by mmacfadden** [orientechnologies/orientdb#6277]
In 2.2.0, you can not import a database that had a default value set for one of the properties. There is a problem in the 2.2.0 version of ODatabasImport.  

Starting on line 989:

```
if (attrib.equals("\"min\""))
        min = value;
      else if (attrib.equals("\"max\""))
        max = value;
      else if (attrib.equals("\"linked-class\""))
        linkedClass = value;
      else if (attrib.equals("\"mandatory\""))
        mandatory = Boolean.parseBoolean(value);
      else if (attrib.equals("\"readonly\""))
        readonly = Boolean.parseBoolean(value);
      else if (attrib.equals("\"not-null\""))
        notNull = Boolean.parseBoolean(value);
      else if (attrib.equals("\"linked-type\""))
        linkedType = OType.valueOf(value);
      else if (attrib.equals("\"collate\""))
        collate = value;
      else if (attrib.equals("\"default-value\""))
        collate = value;
      else if (attrib.equals("\"customFields\""))
        customFields = importCustomFields();
```

Notice that when we get default-value we set collate.  This results in a null pointer exception later on in the code.

I noted that in the develop branch, commit 1b0f490c28a9778a74703a4adde48f7066667140 actually fixes this, even though it was trying to fix another issue.  I am not sure if this has been targeted for a 2.2.x release or not, so I wanted to bring this up in a separate issue to be tracked.

## Expected behavior and actual behavior
The database import should succeed.

## Steps to reproduce the problem
Export a database with any property that has a default value.  You will get a null pointer exception when importing that database.

### Runninng Mode
- [X] Embedded, using PLOCAL access mode
- [X] Embedded, using MEMORY access mode
- [X] Remote

### Misc
- [ ] I have a distributed setup with multiple servers. How many? 
- [ ] I'm using the Enterprise Edition

### OrientDB Version
- [ ] v2.0.x - Please specify last number: 
- [ ] v2.1.x - Please specify last number: 
- [X] v2.2.x - Please specify last number: 

### Operating System
- [ ] Linux
- [X] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

### Java Version
- [ ] 6
- [ ] 7
- [X] 8

**Comment by tglman** [comment]
hi @mmacfadden,

Thank you for so precise reporting, going to fix it quickly 

Regards

**Comment by tglman** [comment]
Hi @mmacfadden,

Yes i confirm is fixed and is going to be released in 2.2.1, can i close it.

Regards

**Comment by mmacfadden** [comment]
@tglman Yes by all means.  I saw that it was fixed.  I mainly reported it so that others would no go through the same debugging process I did for something that was coming soon!
